evm/src/cpu/kernel/asm/hash/blake2b/ops.asm

```diff
@@ -0,0 +1,25 @@
+%macro as_u64
+    %and_const(0xffffffffffffffff)
+%endmacro
```

Comment:
this should probably be in `basic_macros.asm` right next to `%macro as_u32`

Reply:
Good idea!